js/directives/lucca/user-picker.js

```diff
@@ -609,6 +598,34 @@
 			}
 		};
 
+		/**************/
+		/***** ME *****/
+		/**************/
+
+		var initMe = function() {
+			if (init && $scope.displayMeFirst) {
+				getMeAsync().then(function(id) {
+					$scope.myId = id;
+				}, function(message) {
+					errorHandler("GET_ME", message);
+				});
+				init = false;
+			}
+		};
+
+		var getMeAsync = function() {
+			var query = "/api/v3/users/me";
+			var dfd = $q.defer();
+			$http.get(query)
+			.success(function(response) {
+				dfd.resolve(response.data.id);
```

Comment:
if your just interested by the `id`, only ask for the `id`

`/api/v3/users/me?fields=id`